src/server/pps/pretty/pretty.go

```diff
@@ -164,7 +156,7 @@ Parallelism Spec: {{.ParallelismSpec}}
 	CPU: {{ .ResourceSpec.Cpu }}
 	Memory: {{ .ResourceSpec.Memory }} {{end}}
 Inputs:
-{{pipelineInputs .}}
```

Comment:
When people use `inputs` they get translated into the `input` field so no matter what `input` should reflect their pipeline and give them an accurate impression of what's going on.

Although I think I should change the word `Inputs` to `Input` here.